lang/cs/Org.Apache.REEF.Wake/Remote/Impl/StreamingRemoteManager.cs

```diff
@@ -29,27 +29,25 @@ namespace Org.Apache.REEF.Wake.Remote.Impl
     /// <summary>
     /// Manages incoming and outgoing messages between remote hosts.
     /// </summary>
-    /// <typeparam name="T">Message type T. It is assumed to be IWritable</typeparam>
-    [Obsolete("Need to remove Iwritable and use IstreamingCodec. Please see Jira REEF-295 ", false)]
-    public sealed class WritableRemoteManager<T> : IRemoteManager<T> where T : IWritable
+    /// <typeparam name="T">Message type T.</typeparam>
+    public sealed class StreamingRemoteManager<T> : IRemoteManager<T>
     {
-        private static readonly Logger LOGGER = Logger.GetLogger(typeof (WritableRemoteManager<T>));
+        private static readonly Logger Logger = Logger.GetLogger(typeof (StreamingRemoteManager<T>));
 
-        private readonly WritableObserverContainer<T> _observerContainer;
-        private readonly WritableTransportServer<IWritableRemoteEvent<T>> _server;
+        private readonly ObserverContainer<T> _observerContainer;
+        private readonly StreamingTransportServer<IRemoteEvent<T>> _server;
         private readonly Dictionary<IPEndPoint, ProxyObserver> _cachedClients;
-        private readonly IInjector _injector;
+        private readonly IStreamingCodec<IRemoteEvent<T>> _remoteEventCodec;
 
-        /// <summary>
+            /// <summary>
         /// Constructs a DefaultRemoteManager listening on the specified address and
         /// a specific port.
         /// </summary>
         /// <param name="localAddress">The address to listen on</param>
         /// <param name="port">The port to listen on</param>
         /// <param name="tcpPortProvider">Tcp port provider</param>
-        /// <param name="injector">The injector to pass arguments to incoming messages</param>
-        [Obsolete("Use IRemoteManagerFactory.GetInstance() instead.", false)]
```

Comment:
it is ... put back in....While removing Obsolete comments from Writables I mindlessly removed this one too.